packages/playground/src/components/manage_gateway_dialog.vue

```diff
@@ -363,8 +353,18 @@ export default {
 
         updateGrid(grid, { projectName: props.vm ? props.vm.projectName : props.k8s!.projectName });
 
+        let deploymentIps: string[] = [];
+        if (props.vm) {
+          deploymentIps = getDeploymentIps(props.vm);
+        } else if (props.k8s) {
+          deploymentIps = [...props.k8s.masters, ...props.k8s.workers].flatMap(deployment => getDeploymentIps(deployment));
+        }
+
         const { gateways: gws, failedToList } = await loadDeploymentGateways(grid, {
-          filter: () => true,
+          filter:
```

Comment:
might be a good idea to wrap this in a try catch

Reply:
try and catch already existed in the `loadGateways` function, and we have `failedToList` which points to `failedToListGws` , I don't think we need it